scripts/screenshot.config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "uris": {
+    "footer": "http://localhost:6006/?path=/docs/components-footer--footer"
```

Comment:
Nice start.  Are there any other components you want to include in the initial baseline set of screenshots to have visual regression coverage on?

Reply:
They are being added in a cleanup PR